Recrutify/Recrutify.Host/EmailTemplates/Interview_Email.html

```diff
@@ -62,7 +62,7 @@
     <div class="wrapper">
```

Comment:
Rename file to Candidate_Interview_Email

Reply:
+